test/test_store/test_store_triple_store.py

```diff
@@ -1,35 +1,36 @@
-import unittest
+import pytest
 
-from rdflib.term import BNode, Literal
-from rdflib.namespace import RDFS
 from rdflib.graph import Graph
+from rdflib.namespace import RDFS
+from rdflib.term import BNode, Literal
+
+remove_me = (BNode(), RDFS.label, Literal("remove_me"))
+
 
+@pytest.fixture(scope="function")
+def get_store(request):
+    store = Graph(store="default")
+    store.open("store")
+    store.add(remove_me)
 
-class GraphTest(unittest.TestCase):
-    backend = "default"
-    path = "store"
+    yield store
 
-    def setUp(self):
-        self.store = Graph(store=self.backend)
-        self.store.open(self.path)
-        self.remove_me = (BNode(), RDFS.label, Literal("remove_me"))
-        self.store.add(self.remove_me)
+    store.close()
 
-    def tearDown(self):
-        self.store.close()
 
-    def testAdd(self):
-        subject = BNode()
-        self.store.add((subject, RDFS.label, Literal("foo")))
+def test_add(get_store):
+    store = get_store
+    subject = BNode()
+    store.add((subject, RDFS.label, Literal("foo")))
 
-    def testRemove(self):
-        self.store.remove(self.remove_me)
-        self.store.remove((None, None, None))
 
-    def testTriples(self):
-        for s, p, o in self.store:
-            pass
+def test_remove(get_store):
+    store = get_store
```

Comment:
While not wrong, this is not really doing anything, would be better to just use the same name for the fixture and variable in function.

Fine as is though, and would prefer to merge as is, we can fix later, but it just makes for more code.

Reply:
> would be better to just use the same name for the fixture and variable in function.

Some of the fixtures return multiple results.

Reply:
that is true, it is fine like this, not really a massive issue for me, but if you named the fixture test_store in this case you could also just used test_store throughout. Will recheck the rest tonight still.